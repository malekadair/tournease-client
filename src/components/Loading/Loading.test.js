import React from "react";
import toJson from "enzyme-to-json";
import Loading from "./Loading";

describe(`Loading component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a .Loading by default", () => {
    const wrapper = <Loading />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the Loading given props", () => {
    const wrapper = <Loading {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
