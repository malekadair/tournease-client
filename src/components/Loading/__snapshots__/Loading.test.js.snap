// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders a .Loading by default 1`] = `undefined`;

exports[`Loading component renders the Loading given props 1`] = `undefined`;
